keps/976-plain-pods/README.md

```diff
@@ -433,9 +431,12 @@ scheduling. The list of fields to keep are:
   - `preemptionPolicy`
   - `topologySpreadConstraints`
   - `overhead`
-  - `volumes`
   - `resourceClaims`
 
+Note that fields like `env` and `command` can sometimes change among all the pods of a group and
+they don't influence scheduling, so they are safe to skip. `volumes` can influence scheduling, but
+they can be parameterized, like in StatefulSets, so we will ignore them for now.
```

Comment:
Currently, we compare the entire containers to verify if the existing workload matches the desired workload:

https://github.com/kubernetes-sigs/kueue/blob/1ecd79f748af2617111d0bfa1ca848f9e172f4d7/pkg/util/equality/podset.go#L29-L34

So, I'm wondering if we should update envs and commands. WDYT?

Reply:
We could, but I'm not sure about the usefulness.

In Pod groups, it is useful because each Pod might have a slightly different spec. But in Jobs, there is only one template.

But that is a separate discussion nevertheless.

Reply:
It sounds reasonable. I may be able to have discussions about this.
Thanks.